.github/workflows/version.yml

```diff
@@ -14,6 +14,7 @@ on:
     - '*.txt'
     - '*.json'
     - '*.png'
+    - '.github/workflows/version.yml'
```

Comment:
This workflow uses `jitterbit/get-changed-files` (or the input from the manual trigger) to determine which file(s) to sign/upload.
As this is not a file we want to handle, the workflow needs to be adjusted to ignore this (and if this is the only change, none of the current files will be signed/uploaded anyway).

I see two options:
1) We keep it as is and trigger a manual run after merging something that would need a new run without the change of the files.
2) This PR also adjust the workflow to invoke handling of all of the target files if this action changes.

Reply:
Right good point.

The `jitterbit/get-changed-files` action actually causes some warnings anyways. I try to come up with something better.

Reply:
If you go with option 2, do we really want that for when actions/checkout, actions/setup-python and jitterbit/get-changed-files are updated, (or just cosmetic changes to the file for that matter)?

Reply:
Resigning doesn't really hurt, and there might be other reasons to sign again (e.g. adding a new file to the list of signed files).

In the end it isn't all that important I agree. I was not aware of the dispatch option, so I could have used that.

In any case, what I came up with is also a bit of cleanup. I can also drop the trigger on workflow change from that, if preferred.

Reply:
The current approach doesn't work yet when no files need to be signed. Need to handle this specifically.